assets/js/theme/common/url-utils.js

```diff
@@ -45,6 +45,26 @@ const urlUtils = {
 
         return out.substring(1);
     },
+
+    parseQueryParams: (queryData) => {
```

Comment:
Can we please add a test for this method.

Reply:
sure will add it. Thanks for bringing it up :+1